src/domain/common/components/hankeInformationView/HankeInformationView.tsx

```diff
@@ -41,10 +41,12 @@ function InformationViewContentContainer({
 
 function InformationViewMainContent({
   children,
+  className,
 }: {
   children: React.ReactNode | React.ReactNode[];
+  className?: string;
 }) {
-  return <div className={clsx(styles.mainContent, styles.paddingLeft)}>{children}</div>;
+  return <div className={clsx(styles.mainContent, styles.paddingLeft, className)}>{children}</div>;
```

Comment:
The only value ever given to className is `style.mainContent`, already mentioned here. I don't really understand the context, but is the className parameter really needed?

Reply:
Yes, the className parameter is the extra css classes that can be given to that component and it's needed here.